### ISSUE TITLE
Wow!

### ISSUE THREAD

**Issue by chris-short** [dtan4/terraforming#504]
This project is awesome. Thank you for it.

**Comment by BernhardLenz** [comment]
I find it strange that terraform doesn't natively better support importing and re-synching existing infrastructure. The functionality terraform provides natively is just sub-par and this project fills the gap!

**Comment by AnthonyWC** [comment]
> I find it strange that terraform doesn't natively better support importing and re-synching existing infrastructure. The functionality terraform provides natively is just sub-par and this project fills the gap!

Yes it'd be great if Hashicorp can add this project into their core functionality.
